documentation/src/docs/segment-store-service.md

```diff
@@ -66,117 +96,116 @@ Segment Containers are a logical grouping of Segments, and are responsible for a
 
 ### Segment Container Metadata
-The Segment Container Metadata is critical to the good functioning and synchronization of its components. This metadata is shared across all components and it comes at two levels: "Container-wide Metadata" and "per-Segment Metadata". Each serves a different purpose and is described below.
+The Segment Container Metadata is critical to the good functioning and synchronization of its components. This metadata is shared across all components and it comes at two levels: "Container-wide metadata" and "per-Segment metadata". Each serves a different purpose and described below.
 
 #### Container Metadata
 
 Each **Segment Container** needs to keep some general-purpose metadata that affects all operations inside the container:
 
-- **Operation Sequence Number**: The largest sequence number assigned by the _Durable Log_. Every time a new Operation is received and successfully processed by the _Durable Log_, this number is incremented (its value never decreases or otherwise roll back, even if an operation failed to be persisted).
-    - The Operation Sequence Number is guaranteed to be strict-monotonic increasing (no two Operations have the same value, and an Operation will always have a larger Sequence Number than all Operations before it).
+- **Operation Sequence Number**: The largest sequence number assigned by the _Durable Log_. Every time a new operation is received and successfully processed by the _Durable Log_, this number is incremented (its value never decreases or otherwise roll back, even if an operation failed to be persisted).
+    - The operation sequence number is guaranteed to be strict-monotonic increasing (no two operations have the same value, and an operation will always have a larger sequence number than all operations before it).
 - **Epoch**: A number that is incremented every time a successful recovery (Container Start) happens. This value is durably incremented and stored as part of recovery and can be used for a number of cases (a good use is Tier 2 fencing for HDFS, which doesn't provide a good, native mechanism for that).
-- **Active Segment Metadata**: Information about each active Segment (see next section below). A Segment is active if it has had activity (read or write) recently and is currently loaded in memory. If a Segment is not used for a while, or if there are many Segments currently active, a Segment becomes inactive by having its outstanding metadata flushed to Tier 2 Storage and evicted from memory.
+- **Active Segment Metadata**: Provides information about each active Stream Segment. A Segment is active if it has had activity (read or write) recently and is currently loaded in memory. If a Stream Segment is idle for a while, or if there are many Stream Segments currently active, a Stream Segment becomes inactive by having its outstanding metadata flushed to Tier 2 Storage and evicted from memory.
```

Comment:
It doesn't "provide" information, it "keeps" information. Either change it to "Keeps" or leave it as it was before, which is consistent with the rest of the bullets in this section.

Reply:
Accepted Suggestion